Workaround for "invalid application of 'sizeof' to incomplete type" error when the bpf_core_relo struct
is referenced.

diff --git a/src/cc/bcc_btf.cc b/src/cc/bcc_btf.cc
index 7f551ae8..5640cfcd 100644
--- a/src/cc/bcc_btf.cc
+++ b/src/cc/bcc_btf.cc
@@ -33,6 +33,8 @@
 
 namespace btf_ext_vendored {
 
+#include <uapi/linux/bpf.h>
+
 /* The minimum bpf_func_info checked by the loader */
 struct bpf_func_info_min {
         uint32_t   insn_off;
